### DISCUSSION TITLE
Edge case - Validator empy string is a uuid

### DISCUSSION THREAD

**Discussion by kocoten1992** [laravel/framework#39829] [Q&A] [answered]
- Laravel Version: 8.73.1
- PHP Version: 8.0.13
- Database Driver & Version: N/A

### Description:
```php
$validator = Validator::make(['uuid' => ''], ['uuid' => 'uuid']);
$validator->passes(); // true
```
It is suppose to be `false` here ?

**Comment by driesvints** [comment, marked as answer]
You're not requiring the value so the result is correct. Try:

```php
$validator = Validator::make(['uuid' => ''], ['uuid' => 'required|uuid']);
```

Also, please try Discussions first before posting issues.
